grin/src/server.rs

```diff
@@ -122,7 +122,11 @@ impl Server {
 			Seeding::None => {
 				warn!(
 					LOGGER,
-					"No seed configured, will stay solo until connected to"
+					"No seed(s) configured, will stay solo until connected to"
+				);
+				seed.connect_and_monitor(
```

Comment:
I have been testing with nodes that do not necessarily reach out to peer with others - but are themselves treated as a seed (or a known peer).
So - want to make sure this node monitors its own peer list in case somebody else peers with it (and subsequently disappears).